### ISSUE TITLE
More Lua errors on login

### ISSUE THREAD

**Issue by b-morgan** [stpain/guildbook#19]
```
1x Guildbook\Guildbook_ModBlizzUI.lua:65: attempt to index field 'GameTimeFrame' (a nil value)
[string "@Guildbook\Guildbook_ModBlizzUI.lua"]:65: in function `ForceCalendarButton'
[string "@Guildbook\Guildbook_Core.lua"]:182: in function <Guildbook\Guildbook_Core.lua:174>

Locals:
self = <table> {
 BANKFRAME_OPENED = <function> defined @Guildbook\Guildbook_Core.lua:2580
 OnCharacterInventoryRequest = <function> defined @Guildbook\Guildbook_Core.lua:1814
 DebugColours = <table> {
 }
 OnProfileRequest = <function> defined @Guildbook\Guildbook_Core.lua:1718
 ShareWithPlayer = <function> defined @Guildbook\Guildbook_Core.lua:966
 SendGuildCalendarEvents = <function> defined @Guildbook\Guildbook_Core.lua:2207
 HideCalendarButton = <function> defined @Guildbook\Guildbook_ModBlizzUI.lua:60
 UnregisterComm = <function> defined @Questie\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:153
 ScanTradeSkill = <function> defined @Guildbook\Guildbook_Core.lua:1121
 ProfessionNames = <table> {
 }
 TRADE_SKILL_UPDATE = <function> defined @Guildbook\Guildbook_Core.lua:2437
 CHAT_MSG_SYSTEM = <function> defined @Guildbook\Guildbook_Core.lua:2549
 GetInstanceInfo = <function> defined @Guildbook\Guildbook_Core.lua:1473
 FONT_COLOUR = "|cff0070DE"
 CHAT_MSG_WHISPER = <function> defined @Guildbook\Guildbook_Core.lua:2523
 CharacterDataRequest = <function> defined @Guildbook\Guildbook_Core.lua:1916
 player = <table> {
 }
 SendGuildCalendarDeletedEvents = <function> defined @Guildbook\Guildbook_Core.lua:2316
 NEW_RECIPE_LEARNED = <function> defined @Guildbook\Guildbook_Core.lua:2449
 CreateHelperIcon = <function> defined @Guildbook\Guildbook_Core.lua:670
 GUILD_ROSTER_UPDATE = <function> defined @Guildbook\Guildbook_Core.lua:2561
 OptionsInterface = <table> {
 }
 GetGuildMemberGUID = <function> defined @Guildbook\Guildbook_Core.lua:1524
 GetProfessionData = <function> defined @Guildbook\Guildbook_Core.lua:1392
 TrimNumber = <function> defined @Guildbook\Guildbook_Core.lua:760
 ADDON_LOADED = <function> defined @Guildbook\Guildbook_Core.lua:2389
 ScanPlayerContainers = <function> defined @Guildbook\Guildbook_Core.lua:1048
 CreateGuildRosterCache = <function> defined @Guildbook\Guildbook_Core.lua:470
 CalendarWorldEvents = <table> {
 }
 UpdateFonts = <function> defined @Guildbook\Guildbook_Core.lua:691
 RegisterComm = <function> defined @Questie\Libs\AceComm-3.0\AceComm-3.0.lua:60
 NUM_TALENT_ROWS = 7
 SendProfileRequest = <function> defined @Guildbook\Guildbook_Core.lua:1710
 version = "4.8"
 SendCommMessage = <function> defined @Questie\Libs\AceComm-3.0\AceComm-3.0.lua:87
 GetCharacterInventory = <function> defined @Guildbook\Guildbook_Core.lua:1509
 RequestTradeskillData = <function> defined @Guildbook\Guildbook_Core.lua:483
 AvailableLocales = <table> {
 }
 UnregisterAllComm = <function> defined @Questie\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:174
 SetupGuildCalendarFrame = <function> defined @Guildbook\Guildbook_Legacy.lua:262
 SetupGuildBankFrame = <function> defined @Guildbook\Guildbook_Legacy.lua:32
 GetPaperDollStats = <function> defined @Guildbook\Guildbook_Core.lua:805
 OnTalentInfoReceived = <function> defined @Guildbook\Guildbook_Core.lua:1781
 EventFrame = GuildbookEventFrame {
 }
 OnPrivacyReceived = <function> defined @Guildbook\Guildbook_Core.lua:1665
 OnTradeSkillsRequested = <function> defined @Guildbook\Guildbook_Core.lua:1862
 ON_COMMS_RECEIVED = <function> defined @Guildbook\Guildbook_Core.lua:2591
 SendGuildBankCommitRequest = <function> defined @Guildbook\Guildbook_Core.lua:1997
 LoadTime = 1191499.066000
 OnGuildBankCommitReceived = <function> defined @Guildbook\Guildbook_Core.lua:2026
 MakeFrameMoveable = <function> defined @Ad
```

```
1x Guildbook\Guildbook_Core.lua:422: attempt to index field 'config' (a nil value)
[string "@Guildbook\Guildbook_Core.lua"]:422: in function <Guildbook\Guildbook_Core.lua:421>

Locals:
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index field 'config' (a nil value)"
Guildbook = <table> {
 BANKFRAME_OPENED = <function> defined @Guildbook\Guildbook_Core.lua:2580
 OnCharacterInventoryRequest = <function> defined @Guildbook\Guildbook_Core.lua:1814
 DebugColours = <table> {
 }
 OnProfileRequest = <function> defined @Guildbook\Guildbook_Core.lua:1718
 ShareWithPlayer = <function> defined @Guildbook\Guildbook_Core.lua:966
 SendGuildCalendarEvents = <function> defined @Guildbook\Guildbook_Core.lua:2207
 HideCalendarButton = <function> defined @Guildbook\Guildbook_ModBlizzUI.lua:60
 UnregisterComm = <function> defined @Questie\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:153
 ScanTradeSkill = <function> defined @Guildbook\Guildbook_Core.lua:1121
 ProfessionNames = <table> {
 }
 TRADE_SKILL_UPDATE = <function> defined @Guildbook\Guildbook_Core.lua:2437
 CHAT_MSG_SYSTEM = <function> defined @Guildbook\Guildbook_Core.lua:2549
 CRAFT_UPDATE = <function> defined @Guildbook\Guildbook_Core.lua:2443
 GetInstanceInfo = <function> defined @Guildbook\Guildbook_Core.lua:1473
 RegisterComm = <function> defined @Questie\Libs\AceComm-3.0\AceComm-3.0.lua:60
 SetupGuildCalendarFrame = <function> defined @Guildbook\Guildbook_Legacy.lua:262
 FONT_COLOUR = "|cff0070DE"
 CHAT_MSG_WHISPER = <function> defined @Guildbook\Guildbook_Core.lua:2523
 CharacterDataRequest = <function> defined @Guildbook\Guildbook_Core.lua:1916
 player = <table> {
 }
 SendGuildCalendarDeletedEvents = <function> defined @Guildbook\Guildbook_Core.lua:2316
 NEW_RECIPE_LEARNED = <function> defined @Guildbook\Guildbook_Core.lua:2449
 CreateHelperIcon = <function> defined @Guildbook\Guildbook_Core.lua:670
 GUILD_ROSTER_UPDATE = <function> defined @Guildbook\Guildbook_Core.lua:2561
 OptionsInterface = <table> {
 }
 GetGuildMemberGUID = <function> defined @Guildbook\Guildbook_Core.lua:1524
 GetProfessionData = <function> defined @Guildbook\Guildbook_Core.lua:1392
 TrimNumber = <function> defined @Guildbook\Guildbook_Core.lua:760
 ADDON_LOADED = <function> defined @Guildbook\Guildbook_Core.lua:2389
 ScanPlayerContainers = <function> defined @Guildbook\Guildbook_Core.lua:1048
 CreateGuildRosterCache = <function> defined @Guildbook\Guildbook_Core.lua:470
 CalendarWorldEvents = <table> {
 }
 UpdateFonts = <function> defined @Guildbook\Guildbook_Core.lua:691
 scanRosterTicker = <table> {
 }
 NUM_TALENT_ROWS = 7
 SendProfileRequest = <function> defined @Guildbook\Guildbook_Core.lua:1710
 version = "4.8"
 SendCommMessage = <function> defined @Questie\Libs\AceComm-3.0\AceComm-3.0.lua:87
 GetCharacterInventory = <function> defined @Guildbook\Guildbook_Core.lua:1509
 RequestTradeskillData = <function> defined @Guildbook\Guildbook_Core.lua:483
 AvailableLocales = <table> {
 }
 UnregisterAllComm = <function> defined @Questie\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:174
 PlayerMixin = <table> {
 }
 SetupGuildBankFrame = <function> defined @Guildbook\Guildbook_Legacy.lua:32
 GetPaperDollStats = <function> defined @Guildbook\Guildbook_Core.lua:805
 OnTalentInfoReceived = <function> defined @Guildbook\Guildbook_Core.lua:1781
 EventFrame = GuildbookEventFrame {
 }
 OnPrivacyReceived = <function> defined @Guildbook\Guildbook_Core.lua:1665
 OnTradeSkillsRequested = <function> defined @Guildbook\Guildbook_Core.lua:1862
 ON_COMMS_RECEIVED = <function> defined @Guildbook\Guildbook_Cor
```

**Comment by b-morgan** [comment]
Note: This is a fresh install of 4.8. I never had any previous version. It happens on all my characters and repeats itself on each login.

**Comment by stpain** [comment]
both of these should now be resolved in 4.9
